apps/web/src/views/Ifos/components/IfoFoldableCard/IfoPoolCard/ClaimButton.tsx

```diff
@@ -33,7 +33,10 @@ const ClaimButton: React.FC<React.PropsWithChildren<Props>> = ({ poolId, ifoVers
       if (walletIfoData.version === 3) {
         return walletIfoData.contract.write.harvestPool([poolId === PoolIds.poolBasic ? 0 : 1], { account, chain })
       }
-      if (walletIfoData.version === 7 || walletIfoData.version === 8) {
+      if (walletIfoData.version === 7) {
```

Comment:
not important but just a thought. should we make WalletInfoVersion an enum so making these equality checks is cleaner and use switch statement instead. switch statement works kinda like a map so can be better than if statement in some cases. could do something like

```ts
switch(walletIfoData.version) {
case: WalletInfoData.Version1:
   return .....
case WalletInfoData.version2:
   break;
...
...
```

Reply:
Yeah that'll be a cleaner way to do it